src/worker/utils/is_not_order_query_array.ts

```diff
@@ -0,0 +1,5 @@
+import { IOrderQuery } from "@/common/interfaces";
```

Comment:
please remove this code also

Reply:
Those three parts you've requested change are type guard part for origin code to be working as before.
Type-guard function named `isNotOrderQueryArray` narrowing its parameter's type as IOrderQuery, not IOrderQuery[], without those function, it occurs ts error in code below like this.
```typescript
// ../select/index.ts
// line 61
if ((this.query.order.case || isObject(this.query.order.by))) {
// Property 'case' does not exist on type 'IOrderQuery | IOrderQuery[]'.
// Property 'case' does not exist on type 'IOrderQuery[]'.ts(2339)
// Property 'by' does not exist on type 'IOrderQuery | IOrderQuery[]'.
// Property 'by' does not exist on type 'IOrderQuery[]'.ts(2339)
                this.query.order.idbSorting = false; 
                // Property 'idbSorting' does not exist on type 'IOrderQuery | IOrderQuery[]'.
                // Property 'idbSorting' does not exist on type 'IOrderQuery[]'.ts(2339)
            }

// also same in 
// ../select/not_where.ts
// line 7
```
I think with change of `ISelectQuery`'s property `order` from `IOrderQuery` to `IOrderQuery | IOrderQuery[]`, those lines ar e necessary.
Would you give me a favor to teach me why those lines are unnecessary?

Reply:
@yusa-imit have a look at this PR - https://github.com/ujjwalguptaofficial/JsStore/pull/278

only type change & due to type changes - some explicit convertion to type is needed.

Reply:
@ujjwalguptaofficial Thanks. I think your solution for this problem is lot more efficient than mine.
I will close this pr. And also I hope I can use fixed one with #278 in near future.